### DISCUSSION TITLE
Support for postgresql 15

### DISCUSSION THREAD

**Discussion by rjablonovsky** [heterodb/pg-strom#570] [Ideas / Features]
Hello,

What is required to use pg-strom with postgresql 15 eventualy with postgis 3.3? Is there plan to do release of pg-strom for postgresql 15?

Sincerely,
Radovan Jablonovsky

**Comment by rjablonovsky** [comment]
Maybe this not correct question, but does pg-strom and gpu natively or by other means support uint256 or could work with extension pgmp for example?

**Comment by kaigai** [comment]
The GPU-version of PostGIS implementation is independent from PostGIS. So, as long as its function prototype is equivalent, it is available.

Right now, CUDA supports up to `int128_t`, so we have no plan to support `int256` type.
